README.md

```diff
@@ -53,6 +53,18 @@ The `make` command generates the `kernel64.bin` multiboot-compatible ELF file, t
 The `make iso` command takes the `kernel64.bin`, places it in `grub/boot/` directory hierarchy and generates a `boot.iso`
 out of the `grub/` (using `grub/boot/grub/grub.cfg` as a default GRUB config).
 
+#### Fedora
```

Comment:
Does it make sense to detect current distro via /etc/os-release and automagically include corresponding Makefile.local?

Reply:
That would be the next step. With this PR we create the basic infrastructure for custom Make configs. We can later follow up with autodetecting environments.